### PR TITLE
New package: sfeed

### PR DESCRIPTION
**Pull request by kojicomics**

### CODE REVIEW

File: srcpkgs/sfeed/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+short_desc="RSS and Atom parser"
+maintainer="Rubén Santos <kojicomics@cocaine.ninja>"
+license="ISC"
+homepage="https://git.codemadness.org/sfeed"
+
+do_fetch() {
+	git clone git://git.codemadness.org/sfeed ${wrksrc}
+	cd ${wrksrc}
+	git reset --hard ${version}
+}
```
Comment (Vaelatern) [inline srcpkgs/sfeed/template]: Is there no better way?

Comment (kojicomics) [reply srcpkgs/sfeed/template]: I simplified it by using a oneliner. There's no tarball, so we have to clone the last tag version.

Comment (Johnnynator) [reply srcpkgs/sfeed/template]: http://codemadness.org/releases/sfeed/ this does look like tarballs to me

Comment (Vaelatern) [reply srcpkgs/sfeed/template]: Please use the tarballs, @kojicomics

Comment (kojicomics) [reply srcpkgs/sfeed/template]: @Johnnynator Thanks, I will change it

**Pull request merged**